examples/src/main/java/com/algorand/algosdk/example/Main.java

```diff
@@ -19,15 +19,29 @@
 import com.algorand.algosdk.transaction.Transaction;
 import com.algorand.algosdk.util.Encoder;
 
-import java.security.Security;
+import com.squareup.okhttp.HttpUrl;
+
 import java.math.BigInteger;
 
 
+
 public class Main {
 
     public static void main(String args[]) throws Exception {
-        final String ALGOD_API_ADDR = "http://localhost:8080";
-        final String ALGOD_API_TOKEN = "b3ff7b3765bf33c7557a2cd334238eca07578e3ce266cb03fad93d12aab0622e";
+        if (args.length != 2) {
+            System.err.println("Required parameters: ALGOD_API_ADDR ALGOD_API_TOKEN");
+            System.err.println("The parameter values can be found in algod.net and algod.token files within the data directory of your algod install.");
+            System.exit(1);
+        }
+
+        // If the protocol is not specified in the address, http is added.
+        String algodApiAddrTmp = args[0];
+        if (HttpUrl.parse(algodApiAddrTmp) == null) {
```

Comment:
I do not think we want a new 3rd party dependency only for protocol checking. Let's keep it lean and use startsWith()

Reply:
This is not necessarily a new dependency. 
It is already used in src/main/java/com/algorand/algosdk/algod/client/ApiClient.java and the example eventually depends on it. 
What do you think?

Reply:
Did not know that, then it's ok

Reply:
I think your point is still valid. bazel complained! I'm taking your suggestion.